services/remotecluster/send.go

```diff
@@ -121,6 +122,13 @@ func (rcs *Service) sendFrameToRemote(timeout time.Duration, frame *model.Remote
 	req.Header.Set("Content-Type", "application/json")
 
 	resp, err := rcs.httpClient.Do(req.WithContext(ctx))
+	if metrics := rcs.server.GetMetrics(); metrics != nil {
+		if err != nil || resp.StatusCode != http.StatusOK {
+			metrics.IncrementRemoteClusterMsgErrorsCounter(frame.RemoteId, os.IsTimeout(err))
+		} else {
+			metrics.IncrementRemoteClusterMsgSentCounter(frame.RemoteId)
+		}
+	}
```

Comment:
Here I've opted for incrementing counters at the frame level. This will also include pings. Let me know if you prefer something more granular (e.g. differentiate between message types).

Reply:
Frame level makes sense. :+1: